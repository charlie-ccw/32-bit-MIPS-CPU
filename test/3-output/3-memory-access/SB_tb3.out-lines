register_v0_final 0000ff00
