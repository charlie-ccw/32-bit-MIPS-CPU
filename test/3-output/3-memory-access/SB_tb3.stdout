RAM : INIT : Loading RAM contents from test/1-binary/3-memory-access/sb_tb3.txt
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x240300ff
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0xa0030002
TB : INFO : RAM_ACCESS: Store to 0x00000000, data: 0x00000000
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x8c020000
TB : INFO : RAM_ACCESS: Store to 0x00000000, data: 0x0000ff00
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x0000ff00
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x00000008
TB : INFO : RAM_ACCESS: Read from 0xbfc00010, data: 0x00000000
register_v0_final 0000ff00
test/bus_cpu_tb.v:52: $finish called at 350 (1s)
